pkg/client/unversioned/client.go

```diff
@@ -97,6 +98,10 @@ func (c *Client) Secrets(namespace string) SecretsInterface {
 	return newSecrets(c, namespace)
 }
 
+func (c *Client) ConfigDatas(namespace string) ConfigDatasInterface {
```

Comment:
I dislike the name of this method but it is appropriate in terms of the pattern we have.